includes/Checker/With_Shared_Preparations.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Interface WordPress\Plugin_Check\Checker\With_Shared_Preparations
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+/**
+ * Interface for the shared preparations.
+ *
+ * @since n.e.x.t
+ */
+interface With_Shared_Preparations {
+
+	/**
+	 * Get list of preparation classes.
```

Comment:
Per WP coding standards, functions/methods should start with a third-person verb. Also I suggest rephrasing a bit more generic since this should return more than just classes (see other comment).
```suggestion
	 * Gets the list of shared preparations.
```

Reply:
Updated this, thank you!